magento2-nginx/fpm-7.0/README.md

```diff
@@ -63,6 +64,7 @@ The following variables are supported
 
 Variable | Description | Expected values | Default
 --- | --- | --- | ----
+IMAGE_VERSION | The docker image version to use. Version 1 uses the install_magento*.sh scripts which can be hard to customise. Version 2 uses magento_functions.sh and does a temporary database installation during the "build" phase. | 1/2 | 1
```

Comment:
Not if we want to keep backward compatibility with the currently deployed images.